README.md

```diff
@@ -87,11 +121,66 @@ This API should be integrated in your shopping cart and there are two simple ste
   _`-H 'cache-control: no-cache' `_
   _`-H 'Accept: image/png' `_
 
-- **Notes:**
+### Placing a new order
 
-  For any questions please send an email to hello@storydots.app
+- **URL**
 
+  _/order_
 
-## **Example code snippets:**
+- **Method:**
+
+  `POST` - assigns a StoryDots code to your new purchase
```

Comment:
I'd say "order" since we might be confusing "clients" and "purchases" POV

Reply:
agreed, it's changed now